dll/win32/imm32/imm.c

```diff
@@ -14,13 +14,14 @@
 
 WINE_DEFAULT_DEBUG_CHANNEL(imm);
 
-HMODULE ghImm32Inst = NULL; // Win: ghInst
-PSERVERINFO gpsi = NULL; // Win: gpsi
-SHAREDINFO gSharedInfo = { NULL }; // Win: gSharedInfo
-BYTE gfImmInitialized = FALSE; // Win: gfInitialized
+HMODULE ghImm32Inst = NULL; /* The instance of IMM32 */
+PSERVERINFO gpsi = NULL;
+SHAREDINFO gSharedInfo = { NULL };
+BYTE gfImmInitialized = FALSE; /* Is IMM32 initialized? */
 ULONG_PTR gHighestUserAddress = 0;
+LANGID gwKeyboardLangIdCache = 0; /* Cache for GetKeyboardLayoutCP */
+UINT guKeyboardLayoutCPCache = 0; /* Cache for GetKeyboardLayoutCP */
```

Comment:
If those two variables are only meant to be used _**within**_ GetKeyboardLayoutCP and **never** outside of it, it could then be useful to instead move them _inside_ the function and mark them as static.

Reply:
OK. Make them `static`.